### PR TITLE
Set lookup even with no row

### PR DESCRIPTION
**Pull request by biaoli0**
### Issue #:
Fix RN-783 testing issues, see issue links on commit messages.

### CODE REVIEW

**Review by chris-pollard — APPROVED** [review]

File: packages/data-table-server/src/dataTableService/services/DataElementMetadataDataTableService.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ import { yup } from '@tupaia/utils';
 import { DataTableService } from '../DataTableService';
 
 const requiredParamsSchema = yup.object().shape({
-  dataElementCodes: yup.array().of(yup.string().required()).required(),
+  dataElementCodes: yup.array().of(yup.string().required()).default([]),
```
Comment (IgorNadj) [inline packages/data-table-server/src/dataTableService/services/DataElementMetadataDataTableService.ts:13]: Test fail shows this is actually meant to be required?

**Comment by IgorNadj** [comment]
Is Issue 3 fixed by either of these commits?

File: packages/data-table-server/src/dataTableService/services/DataElementMetadataDataTableService.ts (hunk 1 above)
Comment (biaoli0) [reply packages/data-table-server/src/dataTableService/services/DataElementMetadataDataTableService.ts:13]: I suggest we remove the unit test and make it accept dataElementCodes as undefined.

Comment (IgorNadj) [reply packages/data-table-server/src/dataTableService/services/DataElementMetadataDataTableService.ts:13]: Yep sounds good

**Pull request merged**